test/__tests__/data/shopify.json

```diff
@@ -15,29 +15,7 @@
             "created_at": "2023-02-10T12:05:04.402Z"
         },
         "output": {
-            "context": {
-                "library": {
-                    "name": "RudderStack Shopify Cloud",
-                    "version": "1.0.0"
-                },
-                "integration": {
-                    "name": "SHOPIFY"
-                },
-                "topic": "carts_create"
-            },
-            "event": "Cart Create",
-            "integrations": {
-                "SHOPIFY": true
-            },
-            "type": "track",
-            "properties": {
-                "created_at": "2023-02-10T12:05:04.402Z",
-                "id": "shopify_test3",
-                "note": null,
-                "products": [],
-                "token": "shopify_test3",
-                "updated_at": "2023-02-10T12:16:07.251Z"
-            }
+            "error": "event type carts_create not supported"
```

Comment:
Can we absorb the changes made [here](https://github.com/rudderlabs/rudder-transformer/pull/2082) in this PR ? 
CC @krishna2020

Reply:
I am adding some more cases here and there first it is confirm.Then we can see depending on number conflicts
Will that be right approach?

Reply:
Changes are simple:
Move `redisConnector.js` to `src/util/redis` &
Move `redisConnector.test.js` to `src/util/redis` & redis test cases files to `src/util/testdata`

Reply:
Okay

Reply:
done